src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
Oh, i thought it was clear where i'm going with this -

```js
orignalEvent: event.originalEvent
```

I think this is a best course of action.